.circleci/remote_deploy.sh

```diff
@@ -101,9 +101,9 @@ echo "Finished building new images, running run_terraform.sh."
 
 run_on_deploy_box "sudo touch /var/log/deploy_$CIRCLE_TAG.log"
 run_on_deploy_box "sudo chown ubuntu:ubuntu /var/log/deploy_$CIRCLE_TAG.log"
-run_on_deploy_box "source env_vars && echo -e '######\nStarting new deploy for $CIRCLE_TAG\n######' >> /var/log/deploy_$CIRCLE_TAG.log 2>&1"
+run_on_deploy_box "source env_vars && echo -e '######\nStarting new deploy for $CIRCLE_TAG\n######' 2>&1 | tee -a /var/log/deploy_$CIRCLE_TAG.log"
```

Comment:
The output from this script contains secrets so I purposefully prevented it's output from making it CircleCI. However I agree that we can allow the output of building the docker images  to go to CircleCI.